src/mame/adp/stellafr.cpp

```diff
@@ -308,6 +321,12 @@ void stellafr_state::mux_w(uint8_t data)
 		;
 	if (aw2)
 		;
+
+	m_st->write_rts(1);
+	m_st->write_dtr(enanz1);
+	m_st->write_rts(0);
```

Comment:
That's certainly not what the hardware does. `rts` will just be 0 all the time (same for the code below).